### ISSUE TITLE
Attempting to select a column called 'table_name' as a column from a table object results in the table name being returned

### ISSUE THREAD

**Issue by mikeengland** [kayak/pypika#13]
I was trying to run a query to return table information from Vertica. The table showing this information includes a column called 'table_name'. I wanted to access this when running a union command as in the example by specifying the column names against the table I defined in the PyPika Table object, however just the table name is returned. In my query I wanted to include the 'table_name' column of the 'view_columns' table, but 'view_columns' was returned.

Example union from docs:

``` python
provider_a, provider_b = Tables('provider_a', 'provider_b')
q = Query.from_(provider_a).select(
    provider_a.created_time, provider_a.foo, provider_a.bar
) + Query.from_(provider_b).select(
    provider_b.created_time, provider_b.fiz, provider_b.buz
)
```

``` python
from pypika import Query, Tables

view_columns, table_columns = Tables('view_columns', 'columns')

query = Query.from_(view_columns).select(view_columns.table_id, view_columns.table_schema, view_columns.table_name,
                                         view_columns.column_name, view_columns.data_type) + \
        Query.from_(table_columns).select(table_columns.table_id, table_columns.table_schema, table_columns.table_name,
                                          table_columns.column_name, table_columns.data_type).orderby('table_schema', 'table_name')
```

Output:

'SELECT "table_id","table_schema","view_columns","column_name","data_type" FROM "view_columns" UNION SELECT "table_id","table_schema","columns","column_name","data_type" FROM "columns"'

I can get round this issue by specifying the column names as strings for the time being as below:

``` python
    query = Query.from_(view_columns).select('table_id', 'table_schema', 'view_name',
                                             'column_name', 'data_type') + \
            Query.from_(table_columns).select('table_id', 'table_schema', 'table_name',
                                              'column_name', 'data_type')
```

**Comment by z-bowen** [comment]
I've encountered this now too

**Comment by hlmn** [comment]
i got this too, cant use table_name in where clause

```
key_column_usage = Table('key_column_usage')
q = MySQLQuery.from_('key_column_usage').select(key_column_usage.constraint_name, key_column_usage.table_name, key_column_usage.referenced_table_name).where(
        key_column_usage.table_schema == 'mmt-its'
    ).where(
        key_column_usage.referenced_table_name.notnull()
    ).where(
        (key_column_usage.referenced_table_name == 'matakuliah') | (key_column_usage.table_name == 'matakuliah')
    )
```

**Comment by twheys** [comment]
Published 0.14.2
